cs1/tokenbucket.py

```diff
@@ -0,0 +1,29 @@
+import time
+
+
+class TokenBucket:
+
+    def __init__(self, tokens, time_unit):
+        self.tokens = tokens
+        self.time_unit = time_unit
+        self.bucket = tokens
+        self.last_check = time.time()
+
+    def consume(self):
+        if (self.bucket < 1):
+            time.wait(self.time_unit / self.tokens)
```

Comment:
`time.wait` is not a function. Do not send code that has not been run for review; this is equivalent to sending code that doesn't compile.